CREDITS.md

```diff
@@ -17,9 +17,11 @@ written in Rust with a focus on safety and security. Thanks go to:
 
 Contributors to the Firecracker repository:
 
+* Aaqa Ishtyaq <aaqaishtyaq@gmail.com>
 * Aaron Hill <aa1ronham@gmail.com>
 * Aaron O'Mullan <aaron.omullan@gmail.com>
 * Abhijeet Kasurde <akasurde@redhat.com>
+* acatangiu <adrian@parity.io>
```

Comment:
Should we merge this entry with the below? Which email should be the preferred one?

Reply:
I'd vote merge and use new email

Reply:
I think we should maybe leave both there, so as to indicate that different work was done under the umbrella of different companies?

Reply:
I will leave it in this release for now, and we can think of fixing it in main if need be.